.github/workflows/integ-tests-with-security.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [ 11, 17 ]
+        java: [ 11, 17, 21 ]
```

Comment:
@reta  are we going to start supporting 21 in next 2.x version? Lets suppose if 3.0 release is near by can we skip supporting 2.x? How do we make these decisions?

Reply:
> @reta are we going to start supporting 21 in next 2.x version? Lets suppose if 3.0 release is near by can we skip supporting 2.x? How do we make these decisions?

@vamsi-amazon 2.12.0 is going to be bundled with JDK-21, see please https://github.com/opensearch-project/OpenSearch/issues/10334, https://github.com/opensearch-project/OpenSearch/pull/11003. The context for it - JDK-21 is the next LTS and there is new vector APIs that Apache Lucene and OpenSearch benefit from, including plugins. Thank you.

Reply:
> Lets suppose if 3.0 release is near by can we skip supporting 2.x?
I think when 3.0 is out, than 2.x would go to a maintenance mode, as 1.3.x is these days (but fairly speaking, we are just kicking off the discussions to understand when 3.0 could be out). Thank you.